WalletWasabi.Tests/UnitTests/Hwi/DefaultResponseTests.cs

```diff
@@ -118,17 +120,17 @@ public async Task CanCallAsynchronouslyAsync(HwiClient client)
 		[Fact]
 		public async Task OpenConsoleDoesntThrowAsync()
 		{
-			HwiProcessBridge pb = new HwiProcessBridge();
+			var pb = new HwiProcessBridge(new ProcessInvoker());
```

Comment:
Can ProcessInvoker.SendCommandAsync be static?

Reply:
It can. But more interesting question is whether it should be. When you use `static` too much you basically get to this point: https://github.com/zkSNACKs/WalletWasabi/pull/3849/files where it's hard to add new setting.

Yet, if you find `var pb = new HwiProcessBridge(new ProcessInvoker());` too inconvenient, then people typically add a `static` method on `HwiProcessBridge` like `Create()`:

```csharp
public static HwiProcessBridge Create() {
   return new HwiProcessBridge(new ProcessInvoker());
}
```

and it's still versatile and easily testable code.